src/data/DataSet.ts

```diff
@@ -103,7 +103,10 @@ export default class DataSet {
 export class DataGroup {
   public parent?: DataGroup;
 
-  constructor(public readonly title: string, public readonly datasets: (DataSet | DataGroup)[]) {}
+  constructor(
+    public readonly title: string,
+    public readonly datasets: (DataSet | DataGroup)[],
+  ) {}
```

Comment:
this looks like just whitespace changes...  if so, we should exclude this file from the PR.